Tests/registrationTests.js

```diff
@@ -0,0 +1,137 @@
+var homePage = require('../Pages/homePage.js'),
+    itemPage = require ("../Pages/itemPage"),
+    registerPage = require ("../Pages/registerPage"),
+    data = require("../Data/data.js");
+browser.waitForAngularEnabled(false);
```

Comment:
Add new line between "var homePage = require('../Pages/homePage.js'),
    itemPage = require ("../Pages/itemPage"),
    registerPage = require ("../Pages/registerPage"),
    data = require("../Data/data.js");" and browser ...

Please apply this for every test.

Reply:
I applied this for all regression tests, after merging, I will commit changes to smoke test, so that I don't have a conflict